R/te_datastore_csv.R

```diff
@@ -43,6 +43,19 @@ setValidity("te_datastore_csv", function(object) {
   if (length(msg)) msg else TRUE
 })
 
+# Show
+setMethod(
+  "show",
+  c(object = "te_datastore_csv"),
+  function(object) {
+    catn("A TE Datastore CSV object")
+    catn("N:", object@N, "observations")
+    catn("Periods:", object@files$period)
+    catn("Path:", object@path)
+    catn("Template columns:", paste0(colnames(object@template), collapse = ", "))
```

Comment:
```suggestion
    catn("Columns:", paste0(colnames(object@template), collapse = ", "))
```
Users don't need to worry that there is a template